src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -795,10 +794,10 @@ export default Vue.extend({
         .then(res => res.json())
         .then(resultJSON => {
           // check catalogNbr of resultJSON class matches number of course to add
-          resultJSON.data.classes.forEach((resultJSONclass: any) => {
+          resultJSON.data.classes.forEach((resultJSONclass: CornellCourseRosterCourse) => {
             if (resultJSONclass.catalogNbr === number) {
               const course = resultJSONclass;
-              const creditsC = course.credits || course.enrollGroups[0].unitsMaximum;
+              const creditsC = course.enrollGroups[0].unitsMaximum;
```

Comment:
credits is not a field in a course object from roster API. This is probably from a copy-pasta.